test/test_linalg.py

```diff
@@ -4303,8 +4308,10 @@ def check(equation, *operands):
     @dtypes(torch.double, torch.cdouble)
     def test_einsum_random(self, device, dtype):
         def check(equation, *operands):
-            ref = np.einsum(equation, *[op.cpu().numpy() for op in operands])
-            res = torch.einsum(equation, operands)
+            np_operands = [op.cpu().numpy() for op in operands]
+            path = np.einsum_path(equation, *np_operands)[0]
```

Comment:
Add a comment here. 

Should this be testing einsum vs. einsum (without the optimize kwarg) and then einsum_path vs einsum (with the optimize kwarg)?